todo/src/App.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from "react";
+import logo from "./logo.svg";
+import "./App.css";
+import Todos from "./components/Todos";
+import TodoForm from "./components/TodoForm";
+import { connect } from "react-redux";
+import { addTodo } from './actions';
+import { deleteTodo } from './actions';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      todoText: ""
+    };
+  }
```

Comment:
At this point, I think you're safe to start using the React shorthand:

```
class App extends Component {
  constructor(props) {
    super(props);
    this.state = {
      todoText: ""
    };
  }
}
```

becomes...

```
class App extends Component {
  state = {
      todoText: "",
  }
}
```